scm/src/gmtb_scm_output.F90

```diff
@@ -318,8 +430,23 @@ subroutine output_init(scm_state)
   CALL CHECK(NF90_DEF_VAR(NCID=ncid,NAME='init_hour',XTYPE=NF90_FLOAT,VARID=hour_id))
   CALL CHECK(NF90_PUT_ATT(NCID=ncid,VARID=hour_id,NAME="description",VALUES="model initialization hour"))
   CALL CHECK(NF90_PUT_ATT(NCID=ncid,VARID=hour_id,NAME="units",VALUES=""))
-
-
+  
```

Comment:
We could do the same for the 3d tendencies, or?

Reply:
Sure. I could have an array of variable names and descriptions and shorten the code with a loop for sure. At some point, I was hoping to reimplement the output for the SCM though, so I'm not sure that it is worth it right now. I'd like to implement something where the users could specify which variables to write out by their CCPP standard names, implement time-averaging (instead of only instantaneous), etc.